README.md

```diff
@@ -61,6 +61,8 @@ Daily NuGet builds of the project are also available in our MyGet feed:
 
 ### Get started
 
+* [Building, Debugging and Testing on Windows] (https://github.com/dotnet/roslyn/blob/master/docs/contributing/Building,%20Debugging,%20and%20Testing%20on%20Windows.md)
```

Comment:
How do we make a link that isn't directly to the `master` version? Otherwise this will get confusing if you're looking at a different branch (since sometimes branch instructions differ).